docs/src/content/docs/challenges/angular/52-lazy-load-component.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 🟢 Lazy Load a Component
+description: Challenge 52 is about understanding how to lazy load a component in Angular.
+author: lance-finney
+contributors:
+  - tomalaforge
```

Comment:
I don't need to be quoted here, I didn't do anything 😅

Reply:
I think that was in the boilerplate. I'll update it.